solr/configsets/ecommerce/conf/solrconfig.xml

```diff
@@ -25,6 +25,35 @@
   <queryParser name="querqy" class="querqy.solr.QuerqyDismaxQParserPlugin">
     <!-- this param should be skipUnknownRewriters, see https://github.com/querqy/querqy/pull/170 -->
     <bool name="skipUnkownRewriters">true</bool>
+    <lst name="infoLogging">
+      <!--
+        Define a 'sink' named 'responseSink' to which the logging information
+        will be sent:
+      -->
+
+      <lst name="sink">
+        <str name="id">responseSink</str>
+        <str name="class">querqy.solr.ResponseSink</str>
+      </lst>
+
+      <!--
+        Send the logging information from rewriter 'replace' to sink
+        'responseSink':
+      -->
+      <lst name="mapping">
+        <str name="rewriter">replace</str>
```

Comment:
I'm not sure if the replace rewriter logs anything at all but let's leave this in.

Reply:
yeah, we want that as well ;-)